toucan_connectors/google_big_query/google_big_query_connector.py

```diff
@@ -191,20 +256,52 @@ def _bigquery_variable_transformer(variable: str):
         """Add surrounding for parameters injection"""
         return f'@{variable}'
 
+    @cached_property
+    def _bigquery_client(self) -> bigquery.Client:
+        if self.credentials.jwt_token:
+            # We try to instantiate the bigquery.Client with the given jwt-token
+            _session = CustomRequestSession()
+            _session.headers.update(
+                {
+                    'Authorization': f'Bearer {self.credentials.jwt_token}',
+                    'content-type': 'application/json',
+                }
+            )
```

Comment:
should probably be done in `CustomRequestSession.__init__` by passing the jwt token in the constructor

Reply:
[`35fe3a1` (#1287)](https://github.com/ToucanToco/toucan-connectors/pull/1287/commits/35fe3a10de2998af91e2972bda91d0964c8dda0b)